txtdirect.go

```diff
@@ -294,7 +257,7 @@ func Redirect(w http.ResponseWriter, r *http.Request, c Config) error {
 
 		// Trigger fallback when request isn't from `go get`
 		if r.URL.Query().Get("go-get") != "1" {
-			fallback(w, r, rec.Website, rec.Type, "website", http.StatusFound, c)
+			fallback(w, r, "website", http.StatusFound, c)
```

Comment:
Well, since we never allow requests without `go-get` query I think it should be a permanent redirect. Can't remember the specific reason for using 302.

Reply:
We do allow the the request, it just ends up within website. Just curious why we used `200` instead of `302`.

Reply:
Oh ok then. btw it's not `200`, it's 302 right now.